### PR TITLE
autobump: add formulae

### PR DESCRIPTION
**Pull request by dtrodrigues**
- [ ] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [ ] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: .github/autobump.txt [hunk 1]
```diff
@@ -2371,6 +2375,7 @@ vsh
 vte3
 vue-cli
 vultr
+vunnel
```
Comment (chenrui333) [inline .github/autobump.txt:2378]: this cannot be autobumped

Comment (chenrui333) [reply .github/autobump.txt:2378]: see https://github.com/Homebrew/homebrew-core/pull/168254

**Review by chenrui333 — CHANGES_REQUESTED** [review]
I still think it would be good to have autobump part of the formula process, much easier to cross reference.

File: .github/autobump.txt (hunk 1 above)
Comment (dtrodrigues) [reply .github/autobump.txt:2378]: Is there still an issue with autobump for vunnel? The last 5 updates were with `brew bump-formula-pr`.

**Comment by dtrodrigues** [comment]
Recent PRs for the above 5 formulae performed with `brew bump-formula-pr` and not by abot or maintainer associated with the upstream formula:
https://github.com/Homebrew/homebrew-core/pull/172666
https://github.com/Homebrew/homebrew-core/pull/172857
https://github.com/Homebrew/homebrew-core/pull/172667
https://github.com/Homebrew/homebrew-core/pull/172799
https://github.com/Homebrew/homebrew-core/pull/172814

File: .github/autobump.txt (hunk 1 above)
Comment (chenrui333) [reply .github/autobump.txt:2378]: that is the misleading part of the bump-formula-pr, it does not truly reflect the pr status

Comment (chenrui333) [reply .github/autobump.txt:2378]: ie, bump-formula-pr can be done with args to ignore some deps cc @branchvincent

Comment (branchvincent) [reply .github/autobump.txt:2378]: Should be good to autobump, https://github.com/RedHatProductSecurity/cvss/issues/55 was fixed

```
$ brew update-python-resources vunnel
...
==> Updating resource blocks
```

**Comment by SMillerDev** [comment]
> I still think it would be good to have autobump part of the formula process, much easier to cross reference.

What do we need to cross reference?

**Review by SMillerDev — APPROVED** [review]

**Comment by chenrui333** [comment]
> > I still think it would be good to have autobump part of the formula process, much easier to cross reference.
> 
> What do we need to cross reference?

because it is not always autobumpable.

**Review by chenrui333 — APPROVED** [review]

**Pull request merged**